projects/1.20/assets/weapons-of-miracles-epicfight/wom/lang/zh_cn.json

```
@@ -0,0 +1,183 @@
+{
+	"itemGroup.wom.items": "奇迹武装",
+	
+	"enchantment.wom.invigoration" : "精力充沛",
+	"enchantment.wom.invigoration.desc" : "每级加2点最大耐力值",
+
+	"item.wom.golden_monocle" : "黄金单片镜",
+	"item.wom.golden_chrono" : "黄金怀表",
+	"item.wom.golden_mokassin" : "黄金便鞋",
+	"item.wom.golden_kit" : "黄金吊裤带",
+	
+	"item.wom.diamond_crown" : "钻石皇冠",
+	"item.wom.diamond_armbracelet" : "钻石臂环",
+	"item.wom.diamond_legtopseal" : "钻石大腿束带",
+	"item.wom.diamond_legbottomseal" : "钻石小腿束带",
+
+	"item.wom.emerald_earrings" : "绿宝石耳环",
+	"item.wom.emerald_chakra" : "绿宝石后背圈",
+	"item.wom.emerald_tasset" : "绿宝石髋甲",
+	"item.wom.emerald_anklebracelet" : "绿宝石踝环",
+
+	"item.wom.netherite_mask" : "下界合金面具",
+	"item.wom.netherite_manicle" : "下界合金镣铐",
+	"item.wom.netherite_belt" : "下界合金腰带",
+	"item.wom.netherite_chains" : "下界合金锁链",
+
+	"item.wom.agony" : "苦难之刺",
+	"item.wom.agony.tooltip" : "一把轻盈且锋利的长枪，能使出致命的一击。跳跃攻击能够跳越2个方块。",
+	"skill.wom.agony_plunge" : "天坠之殇",
+	"skill.wom.agony_plunge.tooltip" : "\"向上空瞬移，然后对下方的敌人进行极为迅猛的一击。\"\n攻击敌人将为此充能最多3次\n最终伤害将根据能量次数进行倍增。",

```

```suggestion
	"skill.wom.agony_plunge.tooltip" : "“向上空瞬移，然后对下方的敌人进行极为迅猛的一击。”\n攻击敌人将为此充能最多3次。\n最终伤害将根据能量次数进行倍增。",
```

标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。

此外有无句号应与原文一致

---

已修改